src/adaptations/device-layer/include/Weave/DeviceLayer/WeaveDeviceConfig.h

```diff
@@ -103,7 +103,7 @@
  * The maximum size (in bytes) of a debug logging message.
  */
 #ifndef WEAVE_DEVICE_CONFIG_LOG_MESSAGE_MAX_SIZE
-#define WEAVE_DEVICE_CONFIG_LOG_MESSAGE_MAX_SIZE 256
+#define WEAVE_DEVICE_CONFIG_LOG_MESSAGE_MAX_SIZE 512
```

Comment:
Why is this needed for all devices?

Reply:
I'll move this change to 
src/adaptations/device-layer/include/Weave/DeviceLayer/nRF5/WeaveDevicePlatformConfig.h
instead.

Reply:
Moved this to the weave config file in the lock example app.
PR is here: https://github.com/openweave/openweave-nrf52840-lock-example/pull/57